front-end/src/domains/game-view/components/game-board/game-board.component.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
Any base `React` import is no longer needed after React 18.

As long as the `import React from "react";` import is in either main.tsx or App.tsx we don't need it anywhere else 👍